TelCo.ColorCoder/Mapping.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Text;
+using System.Diagnostics;
+using System.Drawing;
+using System.Text;
+
+namespace TelCo.ColorCoder{
+    partial class Program
+    {
+         public override  string ToString()
+        {
+            StringBuilder ReferenceManual = new StringBuilder();
+            int n = 1;
+            for(int i=0; i < colorMapMajor.Length; i++)
+            {
+                for(int j=0; j < colorMapMinor.Length; j++)
+                {
+                    ReferenceManual.Append(n.ToString() + " " + colorMapMajor[i].ToString() + " " + colorMapMinor[j].ToString() );
+                    n++;
+                }
+            }
```

Comment:
there is no formatting between different color pairs 
Print of this would result in 1 single line display